### PR TITLE
Update imports

### LINKED ISSUE #23: Update imports of various bots
The `sys.path.append` should not be used. 
https://github.com/ALLAN-DIP/baseline_bots/blob/ab8e2442e9e4f5984c4a557c4a651ecbbd2e8e5f/src/baseline_bots/bots/dipnet/RealPolitik.py#L8

This type of import should be used instead: https://github.com/ALLAN-DIP/baseline_bots/blob/ab8e2442e9e4f5984c4a557c4a651ecbbd2e8e5f/src/baseline_bots/bots/loyal_bot.py#L8

### PR DESCRIPTION
**Pull request by kkahadze**
Resolves #23

### CODE REVIEW

File: src/baseline_bots/bots/__init__.py [hunk 1]
```diff
@@ -1,10 +0,0 @@
-from baseline_bots.bots.baseline_bot import BaselineBot
```
Comment (trigaten) [inline src/baseline_bots/bots/__init__.py:1]: Why delete these?

**Review by trigaten — CHANGES_REQUESTED** [review]

**Comment by trigaten** [comment]
Fixes #23

File: src/baseline_bots/bots/__init__.py (hunk 1 above)
Comment (kkahadze) [reply src/baseline_bots/bots/__init__.py:1]: Why keep them?

Comment (trigaten) [reply src/baseline_bots/bots/__init__.py:1]: They allow people to do `from baseline_bots import BaselineBot`, etc., so you don't need to write the whole path when you want to import smtg.

Comment (kkahadze) [reply src/baseline_bots/bots/__init__.py:1]: Okay, I'll put them back in

**Review by trigaten — APPROVED** [review]
TYSM for this!

**Pull request merged**